System/Posix/Env/Internal.hsc

```diff
@@ -22,8 +22,11 @@ getCEnviron = nsGetEnviron >>= peek
 foreign import ccall unsafe "_NSGetEnviron"
    nsGetEnviron :: IO (Ptr (Ptr CString))
 #else
-getCEnviron = peek c_environ_p
+getCEnviron = _getCEnviron
 
-foreign import ccall unsafe "&environ"
```

Comment:
Does `capi` not work the same?

Reply:
As far as I can tell, no. `capi` can only be used to import functions and things that behave like functions.